swap_meet/item.py

```diff
@@ -1,2 +1,36 @@
+import uuid
+
+# WAVE 2
 class Item:
-    pass
\ No newline at end of file
+    # default id to none to start
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            # create large unique numbers and make it an interger
+            self.id = uuid.uuid4().int
+        else:
+            # if id already provided, the self.id will just use it
+            self.id = id
+
+        self.condition = condition
+
+    def get_category(self):
+        # we want to see this override in WAVE 5*
+        return "Item"
```

Comment:
👍 Yep. This may seem silly (and in some ways it is), but we'll be overriding this in Wave 5. There are other ways we could investigate doing this, but we would want to be careful that our category value, which is a user-facing piece of business logic, doesn't become intertwined with the class name, since class names are often named with much less user-friendly names.